changelogs/fragments/ansible-test-podman-json-format.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - now makes a better attempt to support podman when calling ``docker images`` and asking for JSON format.
```

Comment:
Side note: I wonder if it'd be reasonable to have some CLI flag for requesting to prefer podman executable over docker explicitly... (Sure it's out of scope here but still it's a topic to discuss)